api4/channel_test.go

```diff
@@ -947,6 +947,11 @@ func TestGetChannelsForTeamForUser(t *testing.T) {
 		channels, resp = Client.GetChannelsForTeamForUser(th.BasicTeam.Id, th.BasicUser.Id, false, "")
 		CheckNoError(t, resp)
 		assert.Equal(t, 5, len(channels))
+
```

Comment:
Please add some tests for `GetChannelsForTeamForUserWithLastDeleteAt` here

Reply:
You caught me :)

Reply:
Oh I remembered now why I didn't add that test. It's because to test the feature properly, I needed to set a time.Sleep, which I didn't like. I can filter with an old time, but then that becomes the same as getting all archived channels. I can do that as a sanity check if you prefer.